kartothek/io/eager.py

```diff
@@ -131,6 +131,14 @@ def read_dataset_as_dataframes(
         load_dataset_metadata=True,
     )
 
+    if len(ds_factory.tables) > 1:
```

Comment:
This will cause the warning to appear twice as you are also raising the warning in read_dataset_as_metapartitions

Reply:
Got it @lr4d, I will make necessary changes.